packages/report-server/src/reportBuilder/transform/parser/functions/context.ts

```diff
@@ -48,9 +48,27 @@ export const orgUnitIdToCode: ContextFunctionConfig = {
   func: ({ getContext }) => (orgUnitId: string) => {
     const { orgUnits } = getContext();
     if (!orgUnits) {
-      throw new Error("Missing dependency 'orgUnits' required by 'orgUnitIdToCode'");
+      throw new Error("Missing dependency 'orgUnits' required by 'orgUnitIdToCode()'");
     }
     const idToCode = reduceToDictionary(orgUnits, 'id', 'code');
     return idToCode[orgUnitId];
   },
 };
+
+export const orgUnitAttribute: ContextFunctionConfig = {
```

Comment:
This is a slight deviation from the naming convention here, which would normally go `orgUnitCodeToAttribute()`. I prefer your more succinct version tho! It should be fairly simple to rename/refactor the others if we wish, though it does involve using a migration... (definitely not saying we should do this now btw)

Reply:
Yeah I thought the deviation was appropriate because this isn't another x -> y conversion function where xToY would make sense, it's more like `getOrgUnitAttibute`. Happy to keep it different like this?

Reply:
Yup, happy to keep it different 👍